cmd/kogito/command/service/build_service.go

```diff
@@ -16,15 +16,17 @@ package service
 
 import (
 	"fmt"
+	"github.com/kiegroup/kogito-cloud-operator/api"
 	"github.com/kiegroup/kogito-cloud-operator/api/v1beta1"
```

Comment:
why keep both?

Reply:
we keep both because `runtimeType` comes from `api` pkg whereas `KogitoBuild` struct comes from `v1beta1`